tests/gdx-tests/src/com/badlogic/gdx/tests/files/ZipFileHandleResolverTest.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.tests.files;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.assets.loaders.ZipFileHandleResolver;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TiledMapRenderer;
+import com.badlogic.gdx.maps.tiled.TmxMapLoader;
+import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
+import com.badlogic.gdx.tests.utils.GdxTest;
+import com.badlogic.gdx.tests.utils.OrthoCamController;
+
+import java.io.IOException;
+import java.util.zip.ZipFile;
+
+public class ZipFileHandleResolverTest extends GdxTest {
+	private static final String ZIP_FILE_PATH = "data/files/map.zip";
+	private static final String MAP_FILE_NAME = "dungeon.tmx";
+	private TiledMap map;
+	private TiledMapRenderer renderer;
+	private OrthographicCamera camera;
+	private OrthoCamController cameraController;
+	private AssetManager assetManager;
+	private BitmapFont font;
+	private SpriteBatch batch;
+
+	@Override
+	public void create () {
+		float w = Gdx.graphics.getWidth();
+		float h = Gdx.graphics.getHeight();
+
+		camera = new OrthographicCamera();
+		camera.setToOrtho(false, (w / h) * 10, 10);
+		camera.zoom = 2f;
+		camera.update();
+
+		cameraController = new OrthoCamController(camera);
+		Gdx.input.setInputProcessor(cameraController);
+
+		font = new BitmapFont();
+		batch = new SpriteBatch();
+
+		try {
+			ZipFile archive = new ZipFile(Gdx.files.internal(ZIP_FILE_PATH).file());
```

Comment:
`FileHandle.file()` is unsupported on the GWT backend:
https://github.com/libgdx/libgdx/blob/5917a01534d7a2aa134e61b7c7975746e5ef19e4/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtFileHandle.java#L102